recipes-bsp/k3-rti-wdt/files/rules

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/make -f
+
+export CC=gcc
+
+override_dh_fixperms:
+	dh_fixperms 
```

Comment:
This should be called automatically. What is the reason for this override?

Reply:
Right, that was imported blindly by taking a deb_debianize output. Will clean up.